images/iqsharp-base/Dockerfile

```diff
@@ -140,9 +140,10 @@ RUN dotnet iqsharp install --user --path-to-tool="$(which dotnet-iqsharp)"
 # Ensure that the necessary NuGet packages are cached.
 ARG EXTRA_NUGET_PACKAGES=
 RUN echo "Adding standard packages..." && \
-    python -c "import qsharp;" && \
+    dotnet new console -n "foo" -lang Q# && \
```

Comment:
Do we want to delete the temporary project after adding the packages or are we ok with it remaining as part of the image?

Reply:
it is removed at the end.